crates/ra_mbe/src/lib.rs

```diff
@@ -169,4 +169,71 @@ impl_froms!(TokenTree: Leaf, Subtree);
             Some(TextRange::from_to(23.into(), 32.into())),
         )
     }
+
+    #[test]
+    fn dont_die_on_serde() {
```

Comment:
It probably doesn't matter, but we should make some callback to `serde`'s license.

Reply:
Not a lawyer, but MIT LICENSE says:

>The above copyright notice and this permission notice
>shall be included in all copies or substantial portions
>of the Software.

I believe this not a substantiational portion :)

Reply:
:+1: